schemas/latest/with-markdown-descriptions/dev-workspace-template-spec.json

```diff
@@ -1,19 +1,59 @@
 {
   "description": "Structure of the workspace. This is also the specification of a workspace template.",
+  "type": "object",
```

Comment:
I checked the differences with the `diffJsons.sh`, and it turns out that the differences are expected are mostly related to one of the following reasons:
- Changes in descriptions in some fields are expected, due to automatic generation of overrides,
- Some fields are not required anymore, due to automatic generation of overrides with a consistent logic vs mandatory fields in overrides
- The complete removal of the ability to override a `custom` component, since it doesn't really make sense.